### PR TITLE
free() pointer to arrow binary in Python

### PR DESCRIPTION
**Pull request by sc1f**
This PR fixes a small issue in Perspective's arrow loading code in Python - previously, the pointer to the malloc'd section for the arrow binary was never explicitly free'd (as it was in Javascript). This leads to the memory allocated for the binary leaking after the arrow is loaded into Perspective.

With testing using `psutil` and 25 updates using `superstore.arrow`, comparing memory usage with/without free() shows a significant impact:

_Without free()_:
- starting `rss`: 115539968 bytes
- ending `rss`: 302710784 bytes

_With free()_:
- starting `rss`: 113532928 bytes
- ending `rss`: 245538816 bytes

Memory usage grows much more slowly when `free()` is added, demonstrating the leak.

### CODE REVIEW

File: python/perspective/perspective/src/table.cpp [hunk 1]
```diff
@@ -193,6 +194,10 @@ std::shared_ptr<Table> make_table_py(t_val table, t_data_accessor accessor,
         _fill_data(data_table, accessor, input_schema, index, offset, limit, is_update);
     }
 
+    if (is_arrow) {
```
Comment (timkpaine) [inline python/perspective/perspective/src/table.cpp:197]: is there any case where `is_arrow && is_delete`?

Comment (timkpaine) [reply python/perspective/perspective/src/table.cpp:197]: e.g. should this match above `    if (is_arrow && !is_delete) {`

Comment (sc1f) [reply python/perspective/perspective/src/table.cpp:197]: There shouldn't be any - remove only takes a list of pkeys, but this whole conditional block is kind of messy. I'll take a look and fix.

Comment (texodus) [reply python/perspective/perspective/src/table.cpp:197]: This logic is incorrect, but so is the fix suggestion - there should be (or will need to be) one case, for `remove()` ... when we get around to fixing this API method generally ...

**Review by texodus — APPROVED** [review]
Looks good!  Thanks for the PR!

**Pull request merged**